data/campaigns/Legend_of_Wesmere/scenarios/chapter5/20_Council_Ruling.cfg

```diff
@@ -23,17 +23,17 @@
         [part]
             background="story/characters/legmir.png"
             scale_background=no
-            story={CAPTION ( _ "Legmir")} + _ "Kalenz, the council has called you here to demand that you apologize for your actions. You took the decision to help the humans at Tath, and hence risk further war with the Orcs; to strike at the orcs north; and last to attack the Saurians. These were decisions to be taken by the council, not by a field commander!"
+            story={CAPTION ( _ "Legmir")} + _ "Kalenz, the Council has called you here to demand that you apologize for your actions. You took the decision to help the humans at Tath, and hence risk further war with the orcs; to strike at the orcs north; and last to attack the saurians. These were decisions to be taken by the council, not by a field commander!"
```

Comment:
Here too. Also, you've left one uncapitalized in this line.

Reply:
As above, and thanks for catching that - will correct it.